integration-tests/jiva/invalidconfig/invalidconfig_test.go

```diff
@@ -0,0 +1,194 @@
+/*
+Copyright 2019 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package invalidconfig
+
+import (
+	"fmt"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	apis "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha1"
+
+	ns "github.com/openebs/maya/pkg/kubernetes/namespace/v1alpha1"
+	pvc "github.com/openebs/maya/pkg/kubernetes/persistentvolumeclaim/v1alpha1"
+	sc "github.com/openebs/maya/pkg/kubernetes/storageclass/v1alpha1"
+	corev1 "k8s.io/api/core/v1"
+	storagev1 "k8s.io/api/storage/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var (
+	// defaultReplicaLabel represents the jiva replica
+	defaultReplicaLabel = "openebs.io/replica=jiva-replica"
+	// defaultCtrlLabel represents the jiva controller
+	defaultCtrlLabel                  = "openebs.io/controller=jiva-controller"
+	openebsProvisioner                = "openebs.io/provisioner-iscsi"
+	accessModes                       = []corev1.PersistentVolumeAccessMode{corev1.ReadWriteOnce}
+	capacity                          = "5G"
+	scObj                             *storagev1.StorageClass
+	pvcObj                            *corev1.PersistentVolumeClaim
+	nsObj                             *corev1.Namespace
+	openebsCASConfigValue             = "- name: ReplicaCount:\n  Value: 3"
+	pvcLabel, replicaLabel, ctrlLabel string
+)
+
+var _ = Describe("TEST INVALID CAS CONFIGURATIONS IN SC", func() {
+	var (
+		namespaceName = "validation-ns1"
```

Comment:
can we have this as `nsName` like others i.e. `scName` `pvcName`

Reply:
Done